src/mainwindow.cpp

```diff
@@ -724,14 +721,6 @@ MainWindow::~MainWindow() {
     // (#1269, may cause an interruption of the shutdown process)
     //    gitCommitCurrentNoteFolder();
 
-    qApp->setProperty("loggingEnabled", false);
-
-    if (showSystemTray) {
-        // if we are using the system tray lets delete the log window so the
```

Comment:
Will this still work?

Reply:
I tried this case especially + a couple of ways to quit and so far nothing crashed.